src/launch.d.ts

```diff
@@ -243,7 +243,16 @@ export interface Settings {
    *
    * Defaults to `webgl`
    */
-  renderMode?: RenderModes
+  renderMode?: RenderModes,
+
+  /**
+   * The delay, in milliseconds, after the key is released before the selected component
+   * gains focus. This delay can be used to create a smoother transition between focused
+   * components or to prevent rapid focus changes.
+   *
+   * Defaults to `50` (ms)
+   */
+  focusHoldTimeout?: number
```

Comment:
I'm in doubt about the name and the description. Maybe `holdTimeout` is more fitting. What the value does is configure how long a user should hold down a key, until we consider it to be in a `hold` state. And when in a hold state Blits will prevent focus being passed around and only does until the key is lifted again.

Reply:
I believe you are describing focus handling in `application.js` which is a bit different logic than the one in `focus.js`.
In the screen capture attached I set `focusHoldTimeout` to `1000 ms`. As you can see this timeout configures when the focus is triggered on the selected component _after_ the button is released.

Do you still want to rename it? Also do you want the same config to control the focus handling in `application.js`?

Reply:
that's correct, they are not exactly the same but they _are_ related I think, in the sense that they should probably hold the same value.

@DouweCnossen can you confirm this from a UX perspective?